core/src/identity/rsa.rs

```diff
@@ -221,8 +300,9 @@ mod tests {
     const KEY2: &'static [u8] = include_bytes!("test/rsa-3072.pk8");
     const KEY3: &'static [u8] = include_bytes!("test/rsa-4096.pk8");
 
+    // TODO: Remove libp2p_core_v026. For compatibility testing only.
```

Comment:
Will remove this before merging. Leaving it for now to ensure I don't introduce a regression.